Tools/autotest/common.py

```diff
@@ -4954,6 +4956,32 @@ def context_pop(self):
             dead_parameters_dict[p[0]] = p[1]
         self.set_parameters(dead_parameters_dict, add_to_context=False)
 
+        if getattr(self, "old_binary", None) is not None:
+            self.stop_SITL()
+            with open(self.binary, "wb") as f:
+                f.write(self.old_binary)
+                f.close()
+            self.start_SITL(wipe=False)
+            self.set_streamrate(self.sitl_streamrate())
+
+    def context_start_custom_binary(self, extra_defines={}):
+        # grab copy of current binary:
+        context = self.context_get()
+        if getattr(context, "old_binary", None) is not None:
+            raise ValueError("Not nestable ATM")
```

Comment:
Please use "at the moment", instead of ATM

Reply:
Done